apis/messages.es6.js

```diff
@@ -50,6 +50,7 @@ export default class Messages extends BaseAPI {
             const constructor = CONSTRUCTORS[data.kind];
             const thing = new constructor(data.data).toJSON();
             if (constructor === CONSTRUCTORS.t4 &&
+                !Array.isArray(thing.replies) &&
```

Comment:
:no_bell: So if there are replies, replies it's {thing: {replies: { data: { children: [] } } } } but if there are none its { thing: { replies: [] } }. That's odd types :/